### PR TITLE
Tile wrap around

### PR DESCRIPTION
**Pull request by brendancol**
Mercator tile sources now have a `wrap_around` flag. 

`wrap_around = True` (default):
screen shot 2015-12-16 at 10 30 35 pm

`wrap_around = False`:
 screen shot 2015-12-16 at 10 34 13 pm

The addition of wrapping to other layers (e.g. `DynamicImageLayer`, `GeoJSONLayer`) will come in subsequent PRs. The colored layer seen on the US is a `DynamicImageLayer`. 

This example is available at `examples/plotting/file/dynamic_map.py`

### CODE REVIEW

File: bokeh/models/tiles.py [hunk 1]
```diff
@@ -58,6 +58,10 @@ class MercatorTileSource(TileSource):
     resolution (plot_units / pixels) of minimum zoom level of tileset projection. None to auto-compute.
     """)
 
+    wrap_around_180 = Bool(default=True, help="""
```
Comment (bryevdv) [inline bokeh/models/tiles.py]: Maybe just `wrap_around`? With the scale mentioned in the help? I'm not aware of any other property names with digits in them. Obviously it works but it's a little incongruous to me.

Comment (brendancol) [inline bokeh/models/tiles.py]: yes i agree on that.  change made and pushed

**Comment by birdsarah** [comment]
what happens to axis labesl when you use wrap around?

**Comment by brendancol** [comment]
The axises are `Range1d` so they will display the actual coordinates (i.e. not 'wrapped').

File: bokeh/models/tiles.py [hunk 2]
```diff
@@ -58,6 +58,10 @@ class MercatorTileSource(TileSource):
     resolution (plot_units / pixels) of minimum zoom level of tileset projection. None to auto-compute.
     """)
 
+    wrap_around = Bool(default=True, help="""
+    flag to indicate whether to load tiles outside of world bounds.
```
Comment (birdsarah) [inline bokeh/models/tiles.py]: Two comments on this docstring:
1) You're not outside of world bounds :D you've just looped round
2) Can you add a note about the limitation regarding the axes, so at least its documented somewhere that displaying the coordinates is not expected to work.

Comment (brendancol) [inline bokeh/models/tiles.py]: Good call. make quick update

**Comment by birdsarah** [comment]
Tested this locally by updating `plotting/file/dynamic_map.py` as follows

````python
tile_source = STAMEN_TONER
tile_source.wrap_around = False
p.add_tile(tile_source)
````

And it worked as expected.

**Comment by bryevdv** [comment]
I can believe we might want to expand or change this some later, but as all the changes are self-contained I will go ahead and merge.

**Pull request merged**